components/browser/menu/src/test/java/mozilla/components/browser/menu/item/WebExtensionBrowserMenuItemTest.kt

```diff
@@ -119,13 +120,13 @@ class WebExtensionBrowserMenuItemTest {
 
         val iconCaptor = argumentCaptor<BitmapDrawable>()
         verify(imageView).setImageDrawable(iconCaptor.capture())
-        Assert.assertEquals(icon, iconCaptor.value.bitmap)
+        assertEquals(icon, iconCaptor.value.bitmap)
 
         verify(imageView).contentDescription = "title"
-        verify(labelView).setText("title")
-        verify(badgeView).setText("badgeText")
+        verify(labelView).text = "title"
+        verify(badgeView).text = "badgeText"
         verify(badgeView).setTextColor(Color.WHITE)
-        verify(badgeView).setBackgroundColor(Color.BLUE)
+        verify(badgeView).background?.setTint(Color.BLUE)
```

Comment:
This one is already testing the use case when we pass a background it needs a bit of work

Reply:
You could take a look this snipped, here the background is getting initialized before it being used :)
Hope it helps!
 
```kotlin
   @Test
    fun bind() {
        val icon: Bitmap = mock()
        val imageView: ImageView = mock()
        val badgeView = spy(TextView(testContext))
        val background : Drawable = mock()
        val labelView: TextView = mock()
        val container = View(testContext)
        val view: View = mock()

        whenever(badgeView.background).thenReturn(background)
        whenever(view.findViewById<ImageView>(R.id.action_image)).thenReturn(imageView)
        whenever(view.findViewById<ImageView>(R.id.action_image)).thenReturn(imageView)
        whenever(view.findViewById<TextView>(R.id.badge_text)).thenReturn(badgeView)
        whenever(view.findViewById<TextView>(R.id.action_label)).thenReturn(labelView)
        whenever(view.findViewById<View>(R.id.container)).thenReturn(container)
        whenever(view.context).thenReturn(testContext)

        val browserAction = Action(
            title = "title",
            loadIcon = { icon },
            enabled = true,
            badgeText = "badgeText",
            badgeTextColor = Color.WHITE,
            badgeBackgroundColor = Color.BLUE
        ) {}

        val action = WebExtensionBrowserMenuItem(browserAction) {}
        action.bind(mock(), view)
        testDispatcher.advanceUntilIdle()

        val iconCaptor = argumentCaptor<BitmapDrawable>()
        verify(imageView).setImageDrawable(iconCaptor.capture())
        assertEquals(icon, iconCaptor.value.bitmap)

        verify(imageView).contentDescription = "title"
        verify(labelView).text = "title"
        verify(badgeView).text = "badgeText"
        verify(badgeView).setTextColor(Color.WHITE)
        verify(background).setTint(Color.BLUE)
    }
```

Reply:
So i'm gonna delete test that i wrote?

Reply:
Ok, done 🎉
Could you please verify now that test working properly?